std/hash/poseidon_bn254.asm

```diff
@@ -107,7 +110,9 @@ machine PoseidonBN254(LASTBLOCK, operation_id) {
     FIRSTBLOCK * (input_in1 - in1) = 0;
     FIRSTBLOCK * (input_cap - cap) = 0;
 
-    (1 - LAST) * (input_in0 - input_in0') = 0;
-    (1 - LAST) * (input_in1 - input_in1') = 0;
-    (1 - LAST) * (input_cap - input_cap') = 0;
+    map(
```

Comment:
How does this work exactly? This returns an array of constraints which means all of them are added, right?

Reply:
Yes, isolated statements can either evaluate to constraints or arrays of constraints. Those constraints are added to the list.